.github/workflows/ci-cd.yml

```diff
@@ -817,16 +790,6 @@ jobs:
         --user
         '${{ env.TOX_VERSION }}'
 
-    - name: Patch tox.ini for Python 3.6 under Windows
-      if: >-
-        runner.os == 'Windows'
-        && matrix.python-version == '3.6'
-      run: >-
-        sed -i
-        's/^package_env\(\s\)\?=.*/package_env = py36-win-dummy/g'
```

Comment:
There's a related hack in `tox.ini`. It can be removed now as well.

Reply:
Done.